server/system/service/user.go

```diff
@@ -388,14 +388,14 @@ func (svc user) Create(ctx context.Context, new *types.User) (u *types.User, err
 			return
 		}
 
-		//add default user's theme
-		new.Meta.Theme = sass.LightTheme
-
 		// Process avatar initials Image
 		if err = svc.generateUserAvatarInitial(ctx, new); err != nil {
 			return
 		}
 
+		//add default user's theme
+		new.Meta.Theme = sass.LightTheme
```

Comment:
I'd imagine this would be better of as

```suggestion
		if new.Meta == nil {
				new.Meta = &types.UserMeta{}
		}
		new.Meta.Theme = sass.LightTheme
```